ftplugin/markdown.vim

```diff
@@ -731,18 +731,18 @@ function! s:MarkdownClearSyntaxVariables()
 endfunction
 
 augroup Mkd
-    autocmd! * <buffer>
-    autocmd BufWinLeave <buffer> mkview!
-    autocmd BufWinEnter <buffer> silent! loadview
-    autocmd BufWinEnter <buffer> call s:MarkdownRefreshSyntax(1)
+    autocmd!
+    autocmd BufWinEnter *.md call s:MarkdownRefreshSyntax(1)
```

Comment:
There are multiple filetypes that count as markdown, there's a list of them in ftdetect/markdown.vim. Why do you want it changed?

Reply:
Yeah. I am not very confident about this change. I changed it because I was getting errors while closing unnamed buffers. And, sometimes non markdown buffers as well. Other markdown extensions can also be added..

Reply:
I don't really get it I will have to think. The intended path is ftdetect -> setfiletype markdown -> run markdown ftplugin -> set augroup for the markdow buffer. There's filetype.lua now in neovim, so the ftdetect can probably be removed, but I don't understand how the autocommand end up being applied to other buffers...